migrations/versions/8558ca0c7f16_auth.py

```diff
@@ -0,0 +1,61 @@
+"""
+auth
+
+Revision ID: 8558ca0c7f16
+Revises: 143432ee0675
+Create Date: 2022-11-15 13:31:52.495417
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '8558ca0c7f16'
+down_revision = '143432ee0675'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.create_table(
+        'auth_group',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('name', sa.String(length=64), nullable=False),
+        sa.PrimaryKeyConstraint('id'),
+        sa.UniqueConstraint('name')
+    )
+    op.create_table(
+        'auth_user',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('external_uuid', postgresql.UUID(), nullable=True),
+        sa.Column('name', sa.String(length=64), nullable=True),
```

Comment:
e-mail will be part of this table as well?

Reply:
Yes, along with some other fields we may need later.